vue-frontend/src/components/home-new/LandingSection.vue

```diff
@@ -47,7 +47,7 @@
             alt="landing-view-image"
           />
         </picture>
-        <div class="tw-flex-2">
+        <div class="tw-flex-2 tw--mt-16 lg:tw--mt-20">
```

Comment:
Instead of margin-top, can't we use `align-items` or `justify` ?

Reply:
No,because to move content upwards we use margin only. align and justify will just move content left-right-center.

Reply:
I think we have to maintain more things with margin. That's why I asked about it. 
For row flex, `align-items` work for `vertical` alignments, and `justify` for `horizontal` and vice versa work in the column flex. Just try it out.

Reply:
okay